units/systemd-timesyncd.service.in

```diff
@@ -12,7 +12,8 @@ ConditionCapability=CAP_SYS_TIME
 ConditionVirtualization=!container
 DefaultDependencies=no
 RequiresMountsFor=/var/lib/systemd/clock
-After=systemd-remount-fs.service systemd-tmpfiles-setup.service systemd-sysusers.service
+Requires=dev-rtc.device
+After=systemd-remount-fs.service systemd-tmpfiles-setup.service systemd-sysusers.service dev-rtc.device
```

Comment:
So far, I did use clock_set_hwclock (like timedated's method_set_time), which in turn writes directly /dev/rtc. But as far as I understand adjtimex with the right flags should achieve the same?